.github/compatibility-test-matrices/v0.1.x/ica-chain-a.json

```diff
@@ -0,0 +1,11 @@
+{
+  "chain-a": ["v0.1.7"],
```

Comment:
I use specific tags here instead of release branches (e.g. `release/v0.1.x`), what do you think, @chatton?

Reply:
the only downside to this is we are only ever testing released versions. It's not a huge deal as this is a demo repo, I'm happy enough to do this if you are okay with potentially not catching issues in icad images until post tag.